tracer/src/Datadog.Trace/Tags.cs

```diff
@@ -229,12 +229,12 @@ public static class Tags
         /// <summary>
         /// A MongoDB query.
         /// </summary>
-        internal const string MongoDbQuery = "mongodb.query";
+        internal const string MongoDbQuery = "db.statement";
 
         /// <summary>
         /// A MongoDB collection name.
         /// </summary>
-        internal const string MongoDbCollection = "mongodb.collection";
+        internal const string MongoDbCollection = "db.mongodb.collection";
```

Comment:
Can you add this change to the PR description?

Reply:
Done thx